### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_CSVModifyColumn_ScreenReader: Voice Over is announcing header of the table as a “blank.”

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5965]
[Check out Keros!](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yMAS; #A11yTCS; #MAS4.2.5; #MAS1.3.1; #A11y_AzureDataStudio; #MacOs; #DesktopApp

**Environment Details:**
Application name: Azure Data Studio 
Version: 1.8.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS : MacOS Mojave 10.14.5

**Additional Details:**
Tool Name: Voice Over
MAS Reference: MAS1.3.1, MAS4.2.5

**Repro Steps:**
1.Install the app Azure Data Studio.
2.Connect to database->navigate to the Data services folder->HDFS Folder-> navigate to CSV files folder->Right click on the files->Select option Create external table from CSV->Click on the "Next" button->Click on next button.
3.Navigate to the table.
4.Observed the announcement of screen reader.

**Actual:**
Screen Reader is only reading out the information about the cell row and column number not the header value. It reads the header as a “blank.”
**Same issue observed with Jaws.**
**similar issue observed in the below pages:**
1.**Summary** - Enable SQL server2019 preview->select table from the database->Create External Table screen should open->Next button->Create Database Master Key screen should open->click on next button->fill all the required details->click on next button->Summary screen should open->Click on next button.
2.**MapDataSrcObjects**- Enable SQL server2019 preview->select table from the database->Create External Table screen should open->Next button->Create Database Master Key screen should open->click on next button->fill all the required details->click on next button.
**Expected:**
Screen Reader should read the information about the header cell values of the table like “Column Name”, “Data Type” and “Allow Nulls.”

**User Impact:** 
Screen Reader user may not get information about the header cell values.

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
**MAS1.3.1-**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**MAS4.2.3-**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={30e4fb40-dcc1-45b4-b714-5e2c28b23ae8}

**Attachment for Reference:**
[5965_[Accessibility]A11y_AzureDataStudio_CSVModifyColumn_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3275035/5965_.Accessibility.A11y_AzureDataStudio_CSVModifyColumn_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by Charles-Gagnon** [comment]
Should be fixed, header is correctly announced in current version of master.
